tools/dbslower.py

```diff
@@ -2,14 +2,19 @@
 #
 # dbslower      Trace MySQL and PostgreSQL queries slower than a threshold.
 #
-# USAGE: dbslower [-v] [-p PID [PID ...]] [-m THRESHOLD] {mysql,postgres}
+# USAGE: dbslower [-v] [-p PID [PID ...]] [-b PATH_TO_BINARY] [-m THRESHOLD] {mysql,postgres}
```

Comment:
I think I'd go with `-x` for executable, or `-l` for library, and not `-b`, which I think we used for buffer sizes in other tools. It's not very important, but it's nice to have consistency across different tools.